migration/migration.go

```diff
@@ -202,6 +205,71 @@ func getCurrentVersion(db *sql.Tx) (int64, error) {
 	return current, nil
 }
 
+// BootstrapWorkItemLinking makes sure the database is populated with the correct work item link stuff (e.g. category and some basic types)
+func BootstrapWorkItemLinking(ctx context.Context, db *gorm.DB, linkCatRepo *models.GormWorkItemLinkCategoryRepository, linkTypeRepo *models.GormWorkItemLinkTypeRepository) error {
```

Comment:
I think it would make sense here to capture the many arguments we pass down here into a "bootstrapContext" struct or something like that (internal to the package). The very long method signatures here are hard to read.